frontend/src/DisplayNameField.js

```diff
@@ -19,9 +20,12 @@ function DisplayNameField({ name, ...props }) {
 
   return (
     <FormControl isInvalid={meta.error && meta.touched}>
+      <FormLabel htmlFor="displayName" fontWeight="bold">
+        Display Name:
```

Comment:
Don't forget the translations! `<Trans>Display Name:</Trans>`

Reply:
:+1: